lua/neoclip/handlers.lua

```diff
@@ -5,16 +5,26 @@ local storage = require('neoclip.storage')
 local settings = require('neoclip.settings').get()
 
 local function should_add(event)
+    if type(settings.length_limit) == "number" then
```

Comment:
```suggestion
    if settings.length_limit then
```
I think this would be fine. it would allow a user to disable it by setting it to `nil` or `false`. You couldn't set it to `0` but why would you do that anyway :)